docs/HOL_BDD-Security-Framework-integration-into-VSTS/README.md

```diff
@@ -0,0 +1,329 @@
+
+**2. Set up BDD-Security Framework on Ubuntu VM** In this task you will install prerequisites for BDD-Security Framework and then install and set up framework itself through a SSH client.
+
+**3. Integrate BDD-Security into VSTS** In this task you will update Continuous Integration and Continuous Deployment definition in VSTS to trigger test runs and then publish test results.
+
+**4. Update BDD-Security to include more tests** In this task you will add more tests to run by BDD-Security framework. You will also learn how to find the exact reason for test failure.
+
+
+
+## HOL:
+
+### Task 1: Create a Ubuntu VM in Azure
+To integrate BDD-Security with VSTS, you need to create a VM where all the tests are going to be run on.
+
+**Step 1.** Open Azure Portal, navigate to the resource group with your environment and click on "Add".
+
+![](media/1.png)
+
+
+**Step 2.** Enter `Ubuntu Server 16.10` into the search field, press "Enter", select `Ubuntu Server 16.10` option and click on "Create".  
+
+![](media/2.png)
+
+
+**Step 3.** Enter "Name", "User name", select "Password" as authentication type, enter "Password" for this VM and confirm it, choose your subscription and click on "OK".
+
+![](media/3.png)
+
+
+**Step 4.** Choose "DS2_V2 Standard" size on the next page and click "Select".   
+
+
+**Step 5.** On the next page, leave everything as defaults and click "OK".
+
+
+**Step 6.** On the final page, just click on "OK".
+
+
+
+### Task 2: Set up BDD-Security Framework on Ubuntu VM
+
+**Step 1.** Navigate to your Resource Group and click on the newly created VM.
+
+![](media/4.png)
+
+
+**Step 2.** On "Overview" page, copy the public IP address.
+
+![](media/5.png)
+
+
+**Step 3.** Open your preferred SSH client and connect to your Ubuntu machine.
+
+![](media/6.png)
+> **Note:**
+> 1) In this lab we use [PuTTY](http://www.putty.org/) ssh client because we are using a Windows machine.
+> 2) Run the following command to initiate the ssh connection if you are using a Unix machine: `ssh <User>@<PublicIp>`
+
+
+**Step 4.** When the security warning will pop up, note the fingerprint of the Ubuntu VM. This fingerprint is required for VSTS setup later on. Click on "Yes" to add fingerprint to your machine and connect to the machine.
+
+![](media/9.png)
+
+**Step 5.** Enter your credentials to your VM.
+
+![](media/7.png)
+
+
+**Step 6.** Download and install Java Development Kit (JDK) 8 on this VM using the following commands:
+
+```
+sudo apt-get update
+sudo apt-get install default-jre -y
+sudo apt-get install default-jdk -y
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+export PATH=$PATH:/usr/lib/jvm/java-8-openjdk-amd64/bin
+```
+> **Note:** Ubuntu 16.10 is shipped with java 8 by default.
+
+
+**Step 7.** Clone [BDD-Security]( https://github.com/continuumsecurity/bdd-security) repository and navigate to it.
+```
+git clone https://github.com/continuumsecurity/bdd-security.git
+cd bdd-security/
+```
```

Comment:
What do you mean by current folder? running all further commands from current folder later on in the lab or cloning repository into the current folder?